mattermost/files/mattermost.service

```diff
@@ -14,6 +12,7 @@ WorkingDirectory=/opt/mattermost
 User=mattermost
 Group=mattermost
 LimitNOFILE=49152
+Environment="MM_INSTALL_TYPE=ubuntu-testing"
```

Comment:
is this the right env variable? the env value needs to change to the correct value right?

Reply:
Correct. Waiting for some feedback @ https://docs.google.com/document/d/1AXr0aip1pC1p62K-abDGX8rdWv9MtX9UiT8TLaz2iUY/edit?disco=AAAAh9_JbUg to finilize this. 

The functionality is implemented

Reply:
awesome, thanks for taking care of this 💪